static/css/restyle/restyle.less

```diff
@@ -1420,13 +1421,18 @@ h3.author .transfer-ownership {
 // happens, so we go back to a non-full-width screenshot then.
 // See: https://github.com/mozilla/addons-server/issues/1983
 @media (max-width: 960px) {
-  .header-bg,
   .restyle .previews {
     margin-left: 0;
     width: 100%;
   }
 }
 
+@media (max-width: 1018px) {
+  .header-bg {
+    margin-left: -29px;
```

Comment:
That’s half the difference between the page width and the min-width of the background, so it makes sure there isn’t a white gap on the left. I’ll add a comment.